includes/admin/wc-admin-functions.php

```diff
@@ -39,6 +39,7 @@ function wc_get_screen_ids() {
 		'edit-product_tag',
 		'profile',
 		'user-edit',
+		'dashboard',
```

Comment:
Including this on the dashboard would work, but it will also add extra overhead from WC scripts and styles, and potentially introduce conflicts. It isn't ideal, but can we move the CSS needed in this widget to a stylesheet only used on dashboard?

Reply:
It is possible, yes. In addition to the styles I've added, I'll also have to take some styles from your redesign of the order listing page, and include those as well, so there would be a bit of duplication.. Sound ok?

Reply:
Yeah thats probably better than loading them all in and you can target the widget only.